packages/compass-saved-aggregations-queries/src/stores/aggregations-queries-items.ts

```diff
@@ -62,22 +60,9 @@ const reducer: Reducer<State, Actions | EditItemActions | DeleteItemActions> = (
         items: newItems,
       };
     }
-    case EditItemActionTypes.EditItemUpdated: {
```

Comment:
We don't need to do this, instead we will simply reload all the queries again from the storage (doing that in edit-item.ts down below).